cmd/podman/images/save.go

```diff
@@ -95,7 +99,17 @@ func save(cmd *cobra.Command, args []string) error {
 		if terminal.IsTerminal(int(fi.Fd())) {
 			return errors.Errorf("refusing to save to terminal. Use -o flag or redirect")
 		}
-		saveOpts.Output = "/dev/stdout"
+		pipePath, cleanup, err = setupPipe()
+		if err != nil {
+			return err
+		}
+		defer func() {
+			errc := cleanup()
+			if err := <-errc; err != nil && err.Error() != "wait canceled" {
```

Comment:
Can you make "wait canceled" into a constant error, rather then a string.  Call it Success.